portal-api/src/v1/helpers/hasAdditionalFiltersStart.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * hasAdditionalFiltersStart
+ * checks to return currentStartPage or startPage set to 0
+ * checks if any filters are selected (apart from default filters)
+ * If sorting object is populated but additional filters are selected, then returns 0 for startPage
+ * If sorting is set or no additional filters, returns currentStartPage
+ * @param {Number} currentStartPage
+ * @param {Object} filters
+ * @param {Object} sort
+ * @returns {Number} startpage
+ */
+const hasAdditionalFiltersStart = (currentStartPage, filters, sort) => {
```

Comment:
Can we rename the function to `computeSkipPosition` please to be more descriptive?